package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "real-time-processing-data-application",
+  "version": "1.0.0",
+  "description": "A Real Time Processing Data Application with AWS architecture",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node app.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tachOrg/Real-Time-Processing-Data-Application.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/tachOrg/Real-Time-Processing-Data-Application/issues"
+  },
+  "homepage": "https://github.com/tachOrg/Real-Time-Processing-Data-Application#readme",
+  "dependencies": {
+    "@aws-sdk/client-dynamodb": "^3.259.0",
+    "aws-sdk": "^2.1304.0",
+    "dotenv": "^16.0.3",
+    "uuid": "^9.0.0"
```

Comment:
do we need this package?

Reply:
We only need "_@aws-sdk/client-dynamodb_" and "_dotenv_" packages in this moment. I'll delete the others packages